js/src/layers/FeatureGroup.js

```diff
@@ -0,0 +1,33 @@
+var widgets = require('@jupyter-widgets/base');
+var _ = require('underscore');
+var L = require('leaflet');
+var layergroup = require('./LayerGroup.js')
+
+
+
+// Model
```

Comment:
Nitpicking here, would you mind removing the extra empty lines that you added? Also, `// Model` and `// View` comments do not add information or readability to the code, so I would remove them too.

Reply:
Yes, I see